src/transformers/models/swin/modeling_tf_swin.py

```diff
@@ -227,7 +227,7 @@ def window_reverse(windows: tf.Tensor, window_size: int, height: int, width: int
     Merges windows to produce higher resolution features.
     """
     x = shape_list(windows)[0]
-    y = tf.cast(height * width / window_size / window_size, tf.int32)
+    y = tf.cast(height * width / (window_size * window_size), tf.int32)
```

Comment:
The previous calculation fails because: 
* `height * width / widow_size` is calculated first, before the final `window_size` division. 
* `height`, `width` are ints, `window_size` is a tensor of type int
* Therefore, the resulting `height * width / widow_size` is a tensor of type float
* This fails when dividing by `window_size` again as the tensors are of different types